theme/material/vaadin-login-overlay-styles.html

```diff
@@ -0,0 +1,356 @@
+<link rel="import" href="../../../vaadin-material-styles/typography.html">
+<link rel="import" href="../../../vaadin-material-styles/color.html">
+<link rel="import" href="../../../vaadin-overlay/vaadin-overlay.html">
+<link rel="import" href="vaadin-login-styles.html">
+
+<dom-module id="vaadin-login-overlay-wrapper-material-styles" theme-for="vaadin-login-overlay-wrapper">
+  <template>
+    <style include="material-color-light material-typography">
+      :host {
+        top: 0;
+        right: 0;
+        bottom: 0;
+        left: 0;
+      }
+
+      [part="card"] {
+        max-width: 100%;
+        box-sizing: border-box;
+        overflow: hidden;
+        display: flex;
+        flex-direction: column;
+      }
+
+      [part="brand"] {
+        box-sizing: border-box;
+        overflow: hidden;
+        flex-grow: 1;
+        flex-shrink: 0;
+        display: flex;
+        flex-direction: column;
+        justify-content: flex-start;
+      }
+
+      [part="brand"] h1 {
+        color: inherit;
+        margin: 0;
+      }
+
+      [part="backdrop"] {
+        background: var(--material-background-color);
+      }
+
+      vaadin-login {
```

Comment:
this selector is not targeting anything (as the login is slotted), and should be removed.
the ` [part="form"] ::slotted(*) {` selector below is a proper one.